src/repos.mjs

```diff
@@ -1,108 +1,13 @@
-const repos = [
-  {
-    upstream: 'fusionjs',
-    name: 'browser-tests',
-  },
-  {
-    upstream: 'fusionjs',
-    name: 'fusion-core',
-  },
-  {
-    upstream: 'fusionjs',
-    name: 'fusion-plugin-react-redux',
-  },
-  {
-    upstream: 'fusionjs',
-    name: 'fusion-plugin-jwt',
-  },
-  {
-    upstream: 'fusionjs',
-    name: 'fusion-plugin-csrf-protection-react',
-  },
-  {
-    upstream: 'fusionjs',
-    name: 'fusion-plugin-csrf-protection',
-  },
-  {
-    upstream: 'fusionjs',
-    name: 'eslint-config-fusion',
-  },
-  {
-    upstream: 'fusionjs',
-    name: 'fusion-react',
-  },
-  {
-    upstream: 'fusionjs',
-    name: 'fusion-test-utils',
-  },
-  {
-    upstream: 'fusionjs',
-    name: 'fusion-cli',
-  },
-  {
-    upstream: 'fusionjs',
-    name: 'fusion-plugin-rpc-redux-react',
-  },
-  {
-    upstream: 'fusionjs',
-    name: 'fusion-plugin-rpc',
-  },
-  {
-    upstream: 'fusionjs',
-    name: 'fusion-rpc-redux',
-  },
-  {
-    upstream: 'fusionjs',
-    name: 'fusion-redux-action-emitter-enhancer',
-  },
-  {
-    upstream: 'fusionjs',
-    name: 'fusion-plugin-node-performance-emitter',
-  },
-  {
-    upstream: 'fusionjs',
-    name: 'fusion-plugin-universal-events-react',
-  },
-  {
-    upstream: 'fusionjs',
-    name: 'fusion-react-async',
-  },
-  {
-    upstream: 'fusionjs',
-    name: 'fusion-plugin-universal-events',
-  },
-  {
-    upstream: 'fusionjs',
-    name: 'fusion-plugin-i18n',
-  },
-  {
-    upstream: 'fusionjs',
-    name: 'fusion-plugin-i18n-react',
-  },
-  {
-    upstream: 'fusionjs',
-    name: 'fusion-plugin-styletron-react',
-  },
-  {
-    upstream: 'fusionjs',
-    name: 'fusion-plugin-react-router',
-  },
-  {
-    upstream: 'fusionjs',
-    name: 'fusion-plugin-browser-performance-emitter',
-  },
-  {
-    upstream: 'fusionjs',
-    name: 'fusion-plugin-universal-logger',
-  },
-  {
-    upstream: 'fusionjs',
-    name: 'fusion-plugin-error-handling',
-  },
-  {
-    upstream: 'fusionjs',
-    name: 'fusion-plugin-service-worker',
-  },
-];
+import api from './api';
 
-export default repos;
+export default async function getRepos() {
+  const {data} = await api.repos.getForOrg({
```

Comment:
Definitely cleaner, but we will now be iterating over stuff like the `rfcs` repos. I think ultimately we might want some sort of listing + tagging mechanism, so we can tag groups of repos to run scripts on.

Reply:
good point! I think in this case we can keep a blacklist rather than a whitelist